src/shared/options/readOptions.test.ts

```diff
@@ -111,6 +111,11 @@ vi.mock("./createOptionDefaults/index.js", () => ({
 	},
 }));
 
+const readPackageData = vi.hoisted(() => vi.fn());
```

Comment:
TIL about `vi.hoisted`, very nice! https://vitest.dev/api/vi.html#vi-hoisted

Elsewhere in this repo I've used `mock*` as a naming convention. I think both are valid and reasonable... but the `mock*` is used everywhere else, so I'll go ahead and make the change.